### PR TITLE
K8s add cluster autoscaler docs

### PR DESCRIPTION
**Pull request by evilnick**
## Done

- Added page for usage of K8s Operator charms
- Added page for Cluster Autoscaler operator

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes

### CODE REVIEW

File: templates/kubernetes/docs/shared/_side-navigation.md [hunk 1]
```diff
@@ -48,6 +48,8 @@
 - [Upgrading](/kubernetes/docs/upgrading)
 - [Storage](/kubernetes/docs/storage)
 - [Scaling](/kubernetes/docs/scaling)
+- [K8s Operator Charms](/kubernetes/docs/scaling)
```
Comment (mtruj013) [inline templates/kubernetes/docs/shared/_side-navigation.md:51]:
```suggestion
- [K8s Operator Charms](/kubernetes/docs/operator-charms)
```

File: templates/kubernetes/docs/shared/_side-navigation.md [hunk 2]
```diff
@@ -48,6 +48,8 @@
...
+- [Cluster autoscaler](/kubermetes/docs/autoscaler)
```
Comment (mtruj013) [inline templates/kubernetes/docs/shared/_side-navigation.md:52]:
```suggestion
- [Cluster autoscaler](/kubernetes/docs/autoscaler)
```

**Review by mtruj013 — APPROVED** [review]
LGTM, just these side navigation paths need to be fixed

**Comment by evilnick** [comment]
@mtruj013 thanks for spotting those - for some reason the demos aren't running or i would have tested better!

**Pull request merged**